src/cli/Microsoft.DotNet.xunit.performance.analysis.cli/project.json

```diff
@@ -33,6 +33,8 @@
     }
   },
   "runtimes": {
-    "win7-x64": {}
+    "win7-x86": {},
```

Comment:
Had a chat with dotnet cli folks, ideally for libraries we don't need to specify the rids but since we need an entryPoint for the runner/analyzer and hence we need to specify the rids. I have filed an issue here to track this https://github.com/dotnet/cli/issues/2913
My other idea was to not specify rids at all and use --infer-runtimes when restoring, but it seems dotnet cli will not be supporting it moving forward.